### PR TITLE
update readme for samples

### PR DESCRIPTION
**Pull request by birsanv**
Update samples readme 

https://issues.redhat.com/browse/ACM-13530

### CODE REVIEW

File: config/samples/usecases/move-all-clusters/move-all-managed-clusters.yaml [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+# After the move-all-managed-clusters restore is applied on Hub2 and has a status of Finished, wait and look for a status Ready 
+# for each managed cluster moved from Hub1 to Hub2. When this happens, the cluster is managed by Hub2.
+# To remove the managed clusters from Hub1, for each managaed cluster, wait until the status is Unknown on Hub1. Only when 
+#  this happens, delete the ManagedCluster global resource on Hub1. This will clean up the
+#  managed cluster resources on Hub1 but since the cluster status it Unknown, it will not run any detach operation - which you don't want to happen.
 
-# You can now resume the BackupSchedules on hub-a.
-# Update the DateProtectionApplication on hub-b to point to hub-b storage location. You can resume now the BackupSchedule on hub-b.
+# You can now resume the BackupSchedules on Hub1.
+# Update the DateProtectionApplication on Hub2 to point to Hub2 storage location. You can resume now the BackupSchedule on Hub2.
```
Comment (tesshuflower) [inline config/samples/usecases/move-all-clusters/move-all-managed-clusters.yaml:25]: Didn't notice this before, typo `DateProtectionApplication` should be `DataProtectionApplication`?

File: config/samples/usecases/move-clusters/move-managed-clusters-step0.yaml [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+# 1. Pause BackupSchedule on Hub1
+# 2. After BackupSchedule is set to paused on Hub1, set this label annotation on the vb-managed-cls-2 ManagedCluster
+#    on Hub1 :  import.open-cluster-management.io/disable-auto-import: ''
+#    This annotation will prevent Hub1 trying to recover vb-managed-cls-2 after is being moved to Hub2
 
-# 3. Pause any BackupSchedule on hub-b
-# 4. Update DateProtectionApplication resource on hub-b to point to the same storage location as hub-a
-# 5. Wait until you see the backups from hub-a being synced up on hub-b
+# 3. Pause any BackupSchedule on Hub2
+# 4. Update DateProtectionApplication resource on Hub2 to point to the same storage location as Hub1
```
Comment (tesshuflower) [inline config/samples/usecases/move-clusters/move-managed-clusters-step0.yaml:16]: typo again with `DataProtectionApplication`

File: config/samples/usecases/move-clusters/move-managed-clusters-step2.yaml [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+# Look for a status Ready for the vb-managed-cls-2 managed cluster on Hub2. At this point, vb-managed-cls-2 
+#  is managed by Hub2.
+# To remove the vb-managed-cls-2 from Hub1, wait until the vb-managed-cls-2 status is Unknown on Hub1. Only when 
+#  this happens, delete the vb-managed-cls-2 ManagedCluster global resource on Hub1. This will clean up all
+#  vb-managed-cls-2 resources on Hub1 but since the cluster status it Unknown, it will not run any detach operation - which you don't want to happen.
 
-# You can now resume the BackupSchedules on hub-a.
-# Update the DateProtectionApplication on hub-b to point to hub-b storage location. You can resume now the BackupSchedule on hub-b.
+# You can now resume the BackupSchedules on Hub1.
+# Update the DateProtectionApplication on Hub2 to point to Hub2 storage location. You can resume now the BackupSchedule on Hub2.
```
Comment (tesshuflower) [inline config/samples/usecases/move-clusters/move-managed-clusters-step2.yaml:13]: typo

File: config/samples/usecases/move-all-clusters/move-all-managed-clusters.yaml (hunk 1 above)
Comment (birsanv) [reply config/samples/usecases/move-all-clusters/move-all-managed-clusters.yaml]: fixed! thanks

File: config/samples/usecases/move-clusters/move-managed-clusters-step0.yaml (hunk 2 above)
Comment (birsanv) [reply config/samples/usecases/move-clusters/move-managed-clusters-step0.yaml]: fixed! thanks

File: config/samples/usecases/move-clusters/move-managed-clusters-step2.yaml (hunk 3 above)
Comment (birsanv) [reply config/samples/usecases/move-clusters/move-managed-clusters-step2.yaml]: fixed! thanks

**Review by tesshuflower — APPROVED** [review]
/lgtm

**Pull request merged**